### DISCUSSION TITLE
Votifier ExtraRewards Commands Running Every X Votes

### DISCUSSION THREAD

**Discussion by IsaiahBloyer** [Zrips/CMI#7876] [Enhancement]
As of my interpretation, current the extra rewards feature runs on a set number of votes. I think a logical and beneficial expansion would to have an argument to have the set of commands ran every x votes. For example, say I have 5 voting sites linked to my server with 24 hour cool downs. I could have where after your 5 daily votes, I give the voter a timed permission through luckperms for the same 24 hours. After the wait, the permission is removed, but the can revote to regain it. Adds onto the incentive to vote

**Comment by mrfloris** [comment]
The features I use in VoteParty and VotingPlugin and a few others are the stuff that lets them vote offline and gives them the rewards once they join the game, and only if they are in certain worlds. So they are in skyblock and vote? no reward, it goes to a queue, and they are offline and join in /wild, or are in skyblock and type /wild? .. then it pays them once they arrive in the wild worlds again.

I also want to use the 'reward players who voted today if we reach x votes, with y commands' and what is suggested by like. 

And hopefully we will figure out how to do these things in CMI, or maybe in the future cmi can expand a bit to offer some more smartypants features so we can expand on how we use it and uninstall the other plugins. 

I don't need all the bloat and super complex features the other plugins have to be honest, but cmi (unless I don't understand how it works) just doesn't really do things yet how I use it (voting wise)
